web-wallet/src/lib/stores/__tests__/walletStore.spec.js

```diff
@@ -111,26 +115,17 @@ describe("walletStore", async () => {
 			expect(get(walletStore)).toStrictEqual(initializedStore);
 		});
 
-		it("should set the gas settings of the wallet based on the settings store values, and update them when they change", async () => {
-			const { gasLimit, gasPrice, ...rest } = get(settingsStore);
-			const newLimit = gasLimit * 3;
-			const newPrice = gasPrice * 3;
+		it("should set the gas settings of the wallet based on the the values that are passed to it", async () => {
```

Comment:
This test is pointless now: your are changing the properties, of course they'll reflect your change.

Reply:
One can never be to sure